scriptmodules/supplementary/splashscreen.sh

```diff
@@ -27,20 +27,41 @@ function _video_exts_splashscreen() {
 }
 
 function depends_splashscreen() {
-    getDepends fbi omxplayer
+    getDepends fbi omxplayer insserv
 }
 
 function install_bin_splashscreen() {
-    cp "$md_data/asplashscreen" "/etc/init.d/"
+    cp -a "$md_data/asplashscreen.sh" "$rootdir/lib/"
 
-    iniConfig "=" '"' /etc/init.d/asplashscreen
+    # remove legacy service
+    [[ -f "/etc/init.d/asplashscreen" ]] && insserv -r asplashscreen && rm -f /etc/init.d/asplashscreen
```

Comment:
This would probably be better done at the configure stage (to match other module behaviour).